### ISSUE TITLE
"Fixnum is deprecated"

### ISSUE THREAD

**Issue by tobireif** [cryptosphere/sysrandom#18]
Hi

I'm getting this error:

```
/usr/local/lib/ruby/gems/2.4.0/gems/sysrandom-1.0.4/lib/sysrandom.rb:49:
warning: constant ::Fixnum is deprecated
```

```
ruby -v
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin15]
```

Tobi

**Comment by tarcieri** [comment]
Fix released in 1.0.5

**Comment by tobireif** [comment]
The warning is gone. Thanks!
